src/frontend/handlers.go

```diff
@@ -52,14 +53,28 @@ var (
 			"renderCurrencyLogo": renderCurrencyLogo,
 		}).ParseGlob("templates/*.html"))
 	plat platformDetails
+	rdb  = redis.NewClient(&redis.Options{
+		Addr:     "localhost:6379", // Cambia esto si Redis está en otra dirección o puerto
+		Password: "",               // Agrega la contraseña si tu Redis la requiere
+		DB:       0,                // Base de datos a usar (0 por defecto)
+	})
 )
 
+func init() {
+	_, err := rdb.Ping(context.Background()).Result()
+	if err != nil {
+		panic(fmt.Sprintf("No se pudo conectar a Redis: %v", err))
+	}
+	fmt.Println("Conexión a Redis exitosa")
+}
```

Comment:
```suggestion
func init() {
    _, err := rdb.Ping(context.Background()).Result()
    if err != nil {
        log.Fatalf("Could not connect to Redis: %v", err) // Use log.Fatalf instead of panic
    }
    fmt.Println("Connected to Redis successfully")
}
```

Reply:
Thank you for the suggestion! I will change it!!